### ISSUE TITLE
shinken forums are spammed

### ISSUE THREAD

**Issue by hahahafr** [naparuba/shinken#1860]
http://forum.shinken-monitoring.org/forums/8-General-Discussion

"Counter-Strike: Global Offensive: Bayern Munich will undertake team of SK Gaming" ...

"What wood preservatives"

yup.

**Comment by bclermont** [comment]
thanks to confirm that shinken is a unmaintained project with a dead project, time to switch to something else

**Comment by k0ste** [comment]
I think we miss @naparuba and @Seb-Solon because of the conflict between them (I do not know this is true or not, but I think so.). Yo can see [alignak](https://github.com/Alignak-monitoring/alignak) as shinken fork. But don't look he have more development and movement.
Sadly, оne of the best project. Works for me without any out of memory, strange exits, etc for 2 years. But looks like dead, yes.

**Comment by olivierHa** [comment]
@bclermont, @k0ste  Shinken isn't an unmaintained project. 
But yes, forums aren't the priority. If you need help, IRC is the way.
It is a community project, so sometimes there will be no commits for a certain period of time, because people are just working on other projects, or are on vacations .. 
You can submit PRs , I will be pleased to merge them ...

**Comment by k0ste** [comment]
@olivierHa, yes you do all work now. @naparuba was the heart of this project, it is very noticeable.

**Comment by olivierHa** [comment]
Don't worry, I am not alone. Just watch @geektophe last PRs about memory management to see that the project is still active.
And @naparuba is still around.

**Comment by mohierf** [comment]
I posted this recently on the Shinken forums: http://forum.shinken-monitoring.org/threads/2034-If-you-have-some-questions-on-the-Shinken-Web-User-Interface and I could not make this message sticky.

And **yes**, the forums are spammed since several weeks up to now. What is worth is that the forums are often an entry point for new comers to the project, and many people are not really aware of IRC channels. 

As far as I remember, the forums were once maintained by @Frescha ? If nobody can maintain now, the best solution would be to inform the visitors to come and meet on the IRC ... it seems a decent minimum thing to do, no ?

**Comment by mohierf** [comment]
@k0ste : unlike what you say, Alignak is a living and evolving Shinken fork... there has been huge code cleaning, documentation and tests effort done by the Alignak team! This effort is still in progress and it takes a lot of time to make the framework as stable as possible. 

 Furthermore, side projects and contributions have been made and are still in progress. If you are interested, please have a look here: https://github.com/Alignak-monitoring-contrib, and connect to the Alignak IRC channel :wink:

**Comment by theHacker** [comment]
I just started playing around with Shinken. Sadly the forums are down completely so when asking Google you have to use Google's cache to get the information.

Just curious: What IRC channel? I did not stumble upon such a thing.

**Comment by dguenault** [comment]
#shinken on irc.freenode.net

Le 13 août 2016 11:11, "Alexander Münch" <notifications@github.com> a
écrit :

I just started playing around with Shinken. Sadly the forums are down
completely so when asking Google you have to use Google's cache to get the
information.

Just curious: What IRC channel? I did not stumble upon such a thing.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/naparuba/shinken/issues/1860#issuecomment-239611333>,
or mute the thread

**Comment by phrampage** [comment]
The only part which is still maintained is web gui 2. What's going on with this project ? is it still an open source solution ? Honestly I don't think so.

**Comment by bclermont** [comment]
@hahahafr you should close the issue and let the dead corpse rot
